app/src/main/java/com/amaze/filemanager/filesystem/usb/SingletonUsbOtg.java

```diff
@@ -0,0 +1,47 @@
+package com.amaze.filemanager.filesystem.usb;
+
+import android.bluetooth.BluetoothClass;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+public class SingletonUsbOtg {
```

Comment:
Prefer name as `UsbOtgSingleton`.

Reply:
That was discussed previously (can't find comment though, so [here is @VishalNehra telling me to correct this name]( https://github.com/TeamAmaze/AmazeFileManager/pull/1172#discussion_r181566394)).

Reply:
That's OK then.